archive.php

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="css/archive.css">
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+</head>
+<body>
+	<?php
+	include 'navbar.php';
+	?>
+	<div id="container">
+		<script>
+		var youtube_videos = ['zh9BTTuOk3g','yXtWhcag5CA', 'bNKRQ7KgtfM', 'joODd099ncs', 'mDEaX8XVsKI', 'AtpzG6q7bj0','02kgVTY3ZuM','Fzfd1tYfjwg','KsXKyUcv4R4',
+		'z3QcBQA3Z3Y','Vk4BL_PoM0g','XbqRY4v08UM','15AUFP0fmrU','4LW8nETl5uQ','RJBwT-kcENI','nvfxcfTLuMA','eQ3kJd0zZxw','69H4Hk5OB8o','6vZUALTQMGo','nsQ-DSSUOi0',
+		'tDGhmJcgZzg','Utf3UQ04Tv8','d6Snm1JWy48'
+							];
+		for(i=0;i<youtube_videos.length;i++){
+			$("#container").append("<div class='col-lg-4 col-md-4 col-sm-6 col-sm-12' id='vid'><iframe frameborder='0' allowfullscreen></iframe></div>");
```

Comment:
Made the changes

On Sun, Sep 6, 2015 at 2:19 AM, Siddharth Kannan <notifications@github.com>
wrote:

> In archive.php
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38819475>
> :
>
> > +<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
> > +<link rel="stylesheet" href="css/archive.css">
> > +<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
> > +</head>
> > +<body>
> > +	<?php
> > +	include 'navbar.php';
> > +	?>
> > +	<div id="container">
> > +		<script>
> > +		var youtube_videos = ['zh9BTTuOk3g','yXtWhcag5CA', 'bNKRQ7KgtfM', 'joODd099ncs', 'mDEaX8XVsKI', 'AtpzG6q7bj0','02kgVTY3ZuM','Fzfd1tYfjwg','KsXKyUcv4R4',
> > +		'z3QcBQA3Z3Y','Vk4BL_PoM0g','XbqRY4v08UM','15AUFP0fmrU','4LW8nETl5uQ','RJBwT-kcENI','nvfxcfTLuMA','eQ3kJd0zZxw','69H4Hk5OB8o','6vZUALTQMGo','nsQ-DSSUOi0',
> > +		'tDGhmJcgZzg','Utf3UQ04Tv8','d6Snm1JWy48'
> > +							];
> > +		for(i=0;i<youtube_videos.length;i++){
> > +			$("#container").append("<div class='col-lg-4 col-md-4 col-sm-6 col-sm-12' id='vid'><iframe frameborder='0' allowfullscreen></iframe></div>");
>
> ping @hitesh4 <https://github.com/hitesh4>
>
> Solve this and update this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38819475>
> .
>